FormalConjectures/Wikipedia/Conway99Graph.lean

```diff
@@ -25,7 +25,12 @@ import FormalConjectures.Util.ProblemImports
 namespace Conway99Graph
 
 --TODO(firsching): Consider using SimpleGraph.IsSRGWith to formulate the conjecture.
+
+open SimpleGraph
```

Comment:
perhaps `open scoped` if you only need this for notation?

Reply:
this was actually needed in a lot of places, and now that it is open we can remove a buch more `SimpleGraph`. done in #1141